spec/controllers/works/multiple_actions_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe WorksController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  let(:multiple_user_pseud) { create(:pseud) }
+  let!(:multiple_works_user) {
```

Comment:
This `!` and the one on line 67 aren't necessary since the user is always invoked before the request.